### PR TITLE
search Bar #75

### PR DESCRIPTION
**Pull request by daniahih**

### CODE REVIEW

File: client/src/components/HomeSerchBar/HomeSearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    fetch('http://localhost:5000/api/search', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ searchTerm }),
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
```
Comment (yishain11) [inline client/src/components/HomeSerchBar/HomeSearchBar.jsx:27]: remove clog

File: client/src/components/HomeSerchBar/HomeSearchBar.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    setSearchTerm(e.target.value);
+  };
+
+  const sendSearchRequest = () => {
+    if (!searchTerm) {
+      setErrorMessage('Please insert text in English, Hebrew, or Arabic');
+      return;
+    }
+
+    fetch('http://localhost:5000/api/search', {
```
Comment (yishain11) [inline client/src/components/HomeSerchBar/HomeSearchBar.jsx:18]: use rtk query

File: client/src/components/HomeSerchBar/HomeSearchBarStyle.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components'
+
+const Wrapped = styled.div`
+    width: 100%;
+    margin: 0 22px 54px 22px;
```
Comment (yishain11) [inline client/src/components/HomeSerchBar/HomeSearchBarStyle.js:5]: use relative units, not px

File: client/src/components/HomeSerchBar/HomeSearchBarStyle.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    .search-container {
+        width: 100%;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .search-input {
+        padding-left: 6px;
```
Comment (yishain11) [inline client/src/components/HomeSerchBar/HomeSearchBarStyle.js:16]: use relative units, not px

File: client/src/components/HomeSerchBar/HomeSearchBarStyle.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        margin-right: 2%;
+        margin-top: -15%;
+        height: 38px;
```
Comment (yishain11) [inline client/src/components/HomeSerchBar/HomeSearchBarStyle.js:19]: use relative units, not px

File: client/src/components/HomeSerchBar/HomeSearchBarStyle.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        width: 272px;
+        font-family: var(--animation-font), sans-serif;
+        font-size: 23px;
```
Comment (yishain11) [inline client/src/components/HomeSerchBar/HomeSearchBarStyle.js:22]: use relative units, not px

File: client/src/components/HomeSerchBar/HomeSearchBarStyle.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        border: 1px solid #000000;
+    }
+
+    button {
+        padding: 0.5px;
+        margin-right: 17px;
```
Comment (yishain11) [inline client/src/components/HomeSerchBar/HomeSearchBarStyle.js:28]: use relative units, not px

**Review by yishain11 — CHANGES_REQUESTED** [review]
in general - use relative units, not px

File: client/src/components/HomeSerchBar/HomeSearchBar.jsx (hunk 1 above)
Comment (daniahih) [reply client/src/components/HomeSerchBar/HomeSearchBar.jsx]: removed

File: client/src/components/HomeSerchBar/HomeSearchBar.jsx (hunk 2 above)
Comment (daniahih) [reply client/src/components/HomeSerchBar/HomeSearchBar.jsx]: done

**Pull request merged**